host-configs/Stanford/sherlock-gcc10-cuda11-ampere.cmake

```diff
@@ -0,0 +1,16 @@
+set(CONFIG_NAME "sherlock-gcc10-cuda11-ampere" CACHE PATH "") 
+
+set(CMAKE_C_COMPILER "/share/software/user/open/gcc/10.1.0/bin/gcc" CACHE PATH "")
+set(CMAKE_CXX_COMPILER "/share/software/user/open/gcc/10.1.0/bin/g++" CACHE PATH "")
+set(CMAKE_Fortran_COMPILER "/share/software/user/open/gcc/10.1.0/bin/gfortran" CACHE PATH "")
```

Comment:
Why did you define `MPI_ROOT` and not some `GCC_ROOT`?

Reply:
Legacy :) These were adapted from my host-configs on other machines, where regular (non-MPI) compiler paths don't always follow a consistent logic, e.g. I had `/path/to/clang-8` and `/path/to/clang++-8`, but `/different/path/to/gfortran-9`.

On Sherlock, since all compilers are from GCC and all have consistent path prefixes, it is indeed a viable option to move these lines into `sherlock-base.cmake` and only have `GCC_ROOT` set outside.